Makefile

```diff
@@ -56,7 +56,7 @@ $(GO):
 	hack/install-go.sh $(BIN_DIR)
 
 $(GINKGO): $(GO) go.mod
-	GOBIN=$$(pwd)/build/_output/bin/ $(GO) install ./vendor/github.com/onsi/ginkgo/ginkgo
+	GOBIN=$$(pwd)/build/_output/bin/ $(GO) install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@v2.1.3
```

Comment:
in the go.mod we have other version already
https://github.com/kubevirt/cluster-network-addons-operator/pull/1260/commits/19add50bd4b44488203f38657103872f329607a1#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R156

i think here it will be better to read the version from the go.mod so it will be auto updated,
you can see how Quique did it on knmstate PR

Reply:
> in the go.mod we have other version already [19add50#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R156](https://github.com/kubevirt/cluster-network-addons-operator/commit/19add50bd4b44488203f38657103872f329607a1#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R156)

I see what you mean here...

> 
> i think here it will be better to read the version from the go.mod so it will be auto updated, you can see how Quique did it on knmstate PR
> 
> [nmstate/kubernetes-nmstate@fbda01d#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R4](https://github.com/nmstate/kubernetes-nmstate/commit/fbda01dfee7e5930e9e630407b5363d863a55751#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R4)

Not sure it's a fail safe idea.. unlike GO-VERSION which appears only once in the go.mod file, the `github.com/onsi/ginkgo/v2` import can appear multiple times, both on `require` and `replace` section..

Reply:
by taking the last occurrence in the file you will take the last one which will be either replace or the require

Reply:
I don't want to rely on me knowing if there is a replace, it might be added in the future by someone else.
order of replace/require is not set.. I'll set a short script, tell me what you think

Reply:
maybe there is a `go mod list package` command ? that say what is the bottom line chosen ?

Reply:
did not find such a tool/command